### PR TITLE
adds pair and list commands

### PR DESCRIPTION
**Pull request by alanbsmith**
#### Why did you need to make these changes?

* adding a structure for endpoints for slack outgoing webhooks to hit an express app.

#### What changes did you make?

* moved route / controller logic to /app/controllers/
* _suggested_ list endpoint should hit the pairs/index route
* _suggested_ pair endpoint should hit the pairs/create route
* moved swifty bot logic to /services/swifty.js

#### Links to documentation of new tech:

* 

#### Obligatory gif

Obligatory Gif

### CODE REVIEW

**Comment by kclowes** [comment]
If you are wanting to merge, it's really helpful for your reviewers if you fill out the PR template :) Thanks!

File: app.js [hunk 1]
```diff
@@ -3,6 +3,11 @@ var logger = require('morgan');
 var cookieParser = require('cookie-parser');
 var bodyParser = require('body-parser');
 
+// controllers
+var controllers = require('./app/controllers/index');
```
Comment (alanbsmith) [inline app.js]: I'm fine with following the `routes` pattern instead, depending on your preference. I'm more familiar with the controller pattern, but it feels like a semantic difference to me. @craftninja

Comment (abrahamsangha) [inline app.js]: It doesn't have to be either `routes` or `controllers`; I'm in favor of keeping the controllers and possibly adding routes in order to simplify `server.js`

File: app/controllers/index.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+var PairsController = require('./pairs');
```
Comment (abrahamsangha) [inline app/controllers/index.js]: this might be better named `controller.js`

File: app.js (hunk 1 above)
Comment (craftninja) [inline app.js]: @abrahamsangha There is no server.js

Comment (abrahamsangha) [inline app.js]: server.js or app.js; same purpose

**Pull request closed without merging**